front/pages/w/[wId]/u/chat/[cId]/index.tsx

```diff
@@ -411,11 +414,18 @@ export function MessageView({
               </div>
             ) : (
               <div className="flex">
-                <img
-                  className="h-8 w-8 rounded-md"
-                  src={user?.image || "https://gravatar.com/avatar/anonymous"}
-                  alt=""
-                />
+                {!readOnly && user?.image ? (
+                  <img
+                    className="h-8 w-8 rounded-md"
+                    src={
+                      user?.image ||
```

Comment:
here user?.image always true (checked above)

Reply:
You're right